<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1080</width>
                <height>800</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="maximumSize">
                <size>
                    <width>1080</width>
                    <height>800</height>
                </size>
            </property>
            <property name="font">
                <font>
                    <family>Forward</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>30</y>
                        <width>1061</width>
                        <height>731</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="spacing">
                        <number>15</number>
                    </property>
                    <item>
                        <widget class="QWidget" name="widget" native="true">
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QPlainTextEdit" name="name">
                                        <property name="enabled">
                                            <bool>true</bool>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>400</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QComboBox" name="genreBox">
                                        <property name="minimumSize">
                                            <size>
                                                <width>200</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>250</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPlainTextEdit" name="yearEdit">
                                        <property name="maximumSize">
                                            <size>
                                                <width>150</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPlainTextEdit" name="duractionEdit">
                                        <property name="maximumSize">
                                            <size>
                                                <width>150</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="findB">
                                        <property name="maximumSize">
                                            <size>
                                                <width>70</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string>Найти</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>
                    </item>
                    <item>
                        <widget class="QTableWidget" name="table"/>
                    </item>
                </layout>
            </widget>
            <widget class="QPushButton" name="addB">
                <property name="geometry">
                    <rect>
                        <x>970</x>
                        <y>770</y>
                        <width>100</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>100</width>
                        <height>0</height>
                    </size>
                </property>
                <property name="maximumSize">
                    <size>
                        <width>100</width>
                        <height>30</height>
                    </size>
                </property>
                <property name="text">
                    <string>Добавить</string>
                </property>
            </widget>
            <widget class="QPushButton" name="updateB">
                <property name="geometry">
                    <rect>
                        <x>810</x>
                        <y>770</y>
                        <width>150</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>150</width>
                        <height>0</height>
                    </size>
                </property>
                <property name="maximumSize">
                    <size>
                        <width>150</width>
                        <height>30</height>
                    </size>
                </property>
                <property name="text">
                    <string>сохранить</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
